dev/cnf/oss_dependencies.maven

```diff
@@ -432,11 +432,18 @@ org.jboss.arquillian.core:arquillian-core-spi:1.3.0.Final
 org.jboss.arquillian.test:arquillian-test-spi:1.3.0.Final
 org.jboss.classfilewriter:jboss-classfilewriter:1.1.2.Final
 org.jboss.logging:jboss-logging:3.4.1.Final
+org.jboss.narayana.arjunacore:arjuna:5.10.5.Final
 org.jboss.narayana.rts:lra-client:5.10.5.Final
+org.jboss.narayana.rts:lra-client:5.10.5.Final
+org.jboss.narayana.rts:lra-coordinator-jar:5.10.5.Final
 org.jboss.narayana.rts:lra-coordinator-jar:5.10.5.Final
 org.jboss.narayana.rts:lra-proxy-api:5.10.5.Final
+org.jboss.narayana.rts:lra-proxy-api:5.10.5.Final
 org.jboss.narayana.rts:lra-service-base:5.10.5.Final
+org.jboss.narayana.rts:lra-service-base:5.10.5.Final
+org.jboss.narayana.rts:narayana-lra:5.10.5.Final
 org.jboss.narayana.rts:narayana-lra:5.10.5.Final
```

Comment:
There seem to be a number of duplicates here?

Reply:
At some point we should make the script that enforces the alphabetical order also check for duplicate lines